File: TrackMyAct.Server/Models/Entities/PlaceEntity.cs
using System.Text.Json.Serialization;

namespace TrackMyAct.Server.Models.Entities;

public class PlaceEntity : Entity
{
    public string Name { get; set; }
    public string Description { get; set; }
    public string Address { get; set; }
    public string AvailableTime { get; set; }

    [JsonIgnore]
    public ICollection<EventEntity> Events { get; set; }
}

File: TrackMyAct.Mobile/composeApp/src/commonMain/kotlin/org/js/tma/dto/PlaceEntity.kt
package org.js.tma.dto

import kotlinx.serialization.Serializable

@Serializable
data class PlaceEntity(
    override var id: Int,
    var name: String,
    var description: String,
    var address: String,
    var availableTime: String,

    var events: List<EventEntity> = emptyList()
) : Entity()